pkg/apis/componentconfig/deep_copy_generated.go

```diff
@@ -1,300 +0,0 @@
-// +build !ignore_autogenerated
```

Comment:
I think a PR went in which included the auto code generation in the build system.